includes/defaults.inc.php

```diff
@@ -440,6 +440,8 @@ function set_debug($debug) {
 // Enable VRFs
 $config['enable_vrf_lite_cisco'] = 1;
 // Enable VRF lite cisco
+$config['enable_route'] = 0;
```

Comment:
Please remove this, with it, it means people have to enable two config options when $config['discovery_modules']['route'] does the same thing.